sdk/tables/aztable/zt_tableRecordedTests.go

```diff
@@ -63,27 +63,27 @@ func recordedTestSetup(t *testing.T, testName string, endpointType EndpointType,
 	assert := assert.New(t)
 
 	// init the test framework
-	context := testframework.NewTestContext(func(msg string) { assert.FailNow(msg) }, func(msg string) { t.Log(msg) }, func() string { return testName })
-	recording, err := testframework.NewRecording(context, mode)
+	context := recording.NewTestContext(func(msg string) { assert.FailNow(msg) }, func(msg string) { t.Log(msg) }, func() string { return testName })
+	recordingInstance, err := recording.NewRecording(context, mode)
```

Comment:
nit: This bight be more idiotmatic and less verboase
```suggestion
	r, err := recording.NewRecording(context, mode)
```